docs/source/new_project.rst

```diff
@@ -53,56 +60,42 @@ can be changed later.
 See Copier's `documentation for changing answers to the question <https://copier.readthedocs.io/en/stable/updating/>`_.
 
 After providing answers to the prompts, Copier will hydrate a project template 
-and save it in the specified location. Additionally Copier will run 
-``git init`` in the new project directory to initialize it as a local repository.
+and save it in the directory you specified.
+
+Create and activate a new environment
+--------------------------------------
 
-Create a new environment
----------------------------
+.. note::
+    This step is optional, but we recommend using virtual environments to better 
+    manage different project's dependencies. 
+    See Python's `description of virtual environments <https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/>`_ 
+    for more details.
 
 If you are using `virtual environments <https://packaging.python.org/en/latest/glossary/#term-Virtual-Environment>`_ 
 create a new environment with your choice of environment tools (virtualenv, 
-conda, etc.) and activate it. 
-This step is optional, but we recommend using virtual environments to better 
-manage different project's dependencies. 
-See Python's `description of virtual environments <https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/>`_ 
-for more details.
+conda, etc.) and activate it.
 
-.. tip::
-  Projects using Python notebooks will need to have ``pandoc`` installed to 
-  convert notebooks to html locally. 
-  Pandoc is `available on conda-forge <https://github.com/conda-forge/pandoc-feedstock>`_, 
-  so conda can be a convenient way to keep these dependencies grouped together.
-
-Install your new package
-----------------------------------------------------
+Prepare your project
+---------------------
 
-Go to the new package directory and install the newly created python package.
-Use ``pip`` to install both the standard set of dependencies as well as the 
-``[dev]`` dependencies.
-
-.. note::
-  Depending on your system you may not need the single quotes around ``'.[dev]'``.
+Once your virtual environment has been created and activated run the following
+script in your new project directory.
 
 .. code-block:: bash
 
-    >> cd <new/project/directory>
-    >> pip install -e .
-    ...
-    Lots of output
-    ...
+    >> bash ./tmp/_prepare_project.sh
```

Comment:
I know we agreed on this and I accept it. It looks a little weird to my eye to have it in tmp. That said I do not have a better idea.

Reply:
Gotcha, I can reconsider.

Reply:
I'm thinking that we just move it out into the base directory, and let people check it into the repository. It can act as a script to configure someone else's environment when they clone the code. So perhaps there's more value in it than I was initially thinking. 

What do you think of that?

Reply:
I've moved it out of tmp, into the base directory. And also renamed the file `.prepare_project.sh`. As stated above, perhaps, as you pointed our during out chat, there would be value in having this as almost a setup script for someone cloning the repo and getting their environment setup.